x-pack/solutions/observability/plugins/apm/common/agent_configuration/setting_definitions/general_settings.ts

```diff
@@ -284,7 +317,15 @@ export const generalSettings: RawSettingDefinition[] = [
       defaultMessage:
         'When recording, the agent instruments incoming HTTP requests, tracks errors, and collects and sends metrics. When set to non-recording, the agent works as a noop, not collecting data and not communicating with the APM Server except for polling for updated configuration. As this is a reversible switch, agent threads are not being killed when set to non-recording, but they will be mostly idle in this state, so the overhead should be negligible. You can use this setting to dynamically control whether Elastic APM is enabled or disabled.',
     }),
-    excludeAgents: ['nodejs', 'rum-js', 'js-base'],
+    excludeAgents: [
+      'nodejs',
+      'rum-js',
+      'js-base',
+      'opentelemetry/dotnet/elastic',
```

Comment:
@jackshirazi Just discussed with @hegerchr and we're wondering whether it's a good idea in general to reuse config options from the classic agents for EDOT SDKs, vs. introducing explicit new config options for those.

For example the description above is slightly Elastic APM / APM Server specific.

Reply:
We can have the same key, for example `recording` defined twice, one with Elastic APM specific description excluding EDOT agents, and another with a description more focused on EDOT agents and add them on the include agents list.

Reply:
that's useful! So do we even need the separate `edot_sdk_settings.ts` file?

Reply:
You don't need to have it on a different file, you just need to exclude EDOT agents from the current `recording` key and include them on the new `recording` key. 

@AlexanderWert / @jackshirazi / @hegerchr let me know what's the copy you want for the EDOT agents and I make the changes.

Reply:
Consensus decision is to use new (different) option names/keys for otel agents. Names have not yet been defined. We may as well use the separate edot file to keep it all distinct and more manageable

Reply:
I'm following up on naming, I should update the PR late tomorrow

Reply:
So this one should be changed back again to exclude all the EDOT agents including the java one, since we're using new options for the EDOT agents